packages/utilities/src/scroll.scss

```diff
@@ -0,0 +1,3 @@
+.scrollDisabled {
```

Comment:
Yes, I understand that, and hashes is good choice. 

I just want to say, that when I'm see `scrollDisabled_hash` class at the `body` it's not fully clear what is this and who add this (ex. is that Office UI or other lib that I'm using). `ms-scroll__body--disabled_hash` I think looks logically better.